include/types.h

```diff
@@ -57,7 +57,8 @@ struct SettingSpec
         Float,
         Boolean,
         String,
-        Palette
+        Palette,
+        Color

```

Comment:
You're well on your way to a complete OLEAUTO type spec!

Reply:
Haha, why did I have a feeling you would bring that up?